polaris.shopify.com/scripts/get-props/src/get-props.ts

```diff
@@ -17,6 +17,15 @@ const compilerOptions = {
   module: ts.ModuleKind.CommonJS,
 };
 
+const deprecatedComponents = [
+  'DisplayText',
+  'Heading',
+  'Subheading',
+  'Caption',
+  'TextStyle',
+  'VisuallyHidden',
+];
```

Comment:
This is fine for now but I would prefer us to not hard code an array and get the `deprecated` value from:
```
status:
  value: Deprecated
```

Would this even run if the components don't exist?!

Reply:
I originally wanted to use `status` but I wasn't sure if us not showing the prop table from all deprecated components and potentially before there's replacement to support functionality would be something we'd want to do. If that's the case, then I'm happy to refactor this 👍